doc/guides/building-node-with-ninja.md

```diff
@@ -22,27 +25,21 @@ The bottom line will change while building, showing the progress as
 `[finished/total]` build steps. This is useful output that `make` does not
 produce and is one of the benefits of using Ninja. Also, Ninja will likely
 compile much faster than even `make -j4` (or
-`-j<number of processor threads on your machine>`).
+`-j<number of processor threads on your machine>`), you can still pass the
+number of processes to run for [Ninja][] using the environment variable `JOBS`,
+and this will be the equivalent to the `-j` parameter in the regular `make`:
 
-## Considerations
-
-Ninja builds vary slightly from `make` builds. If you wish to run `make test`
-after, `make` will likely still need to rebuild some amount of Node.js.
-
-As such, if you wish to run the tests, it can be helpful to invoke the test
-runner directly, like so:
-`tools/test.py --mode=release message parallel sequential -J`
-
-## Alias
-
-`alias nnode='./configure --ninja && ninja -C out/Release && ln -fs
-out/Release/node node'`
+```bash
+$ JOBS=12 make
+```
 
 ## Producing a debug build
 
-The above alias can be modified slightly to produce a debug build, rather than a
+Is possible to create an alias to produce a debug build, rather than a
```

Comment:
Is -> It's ?

Reply:
Thanks for reviewing, just addressed your feedback